src/stories/accidenz_commons.stories.js

```diff
@@ -1,4 +1,4 @@
-export default { title: 'Accidenz Commons' }
+export default { title: "Accidenz Commons" };
```

Comment:
Redundant change. Undo it.
```suggestion
export default { title: 'Accidenz Commons' };
```